x/datadeal/keeper/keeper.go

```diff
@@ -72,7 +97,7 @@ func (k Keeper) IterateClosedDataVerificationQueue(ctx sdk.Context, endTime time
 	defer iter.Close()
 
 	for ; iter.Valid(); iter.Next() {
-		dealID, dataHash := types.SplitDataVerificationQueueKey(iter.Key())
+		_, dealID, dataHash, _ := types.SplitDataQueueKey(iter.Key())
```

Comment:
Isn't there any chance getting error?
I think we need to handling error.

Reply:
Yes, I thought in this function, there has no need to use a endTime so I didn't handle error.
But I think it need to handle error because the unexpected error could be occured.